src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,31 +9,76 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+ }
   componentWillMount() {
     let { setCurrentPursuance, match, currentPursuanceId } = this.props;
     currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
     setCurrentPursuance(currentPursuanceId);
+    ReactModal.setAppElement('body');
```

Comment:
@greg-han Are you sure this is necessary? And can't you set a more specific element within `body`?

Reply:
It's mainly meant for screen readers (accessability).
If I remove it by using "ariaHideApp=false" then the console error tells me that a screen reader may read the body of the pursuance page rather than the modal.